Source/WebKit/Shared/LogStream.h

```diff
@@ -40,16 +40,22 @@ namespace WebKit {
 
 class LogStream : public IPC::StreamMessageReceiver {
 public:
-    LogStream() = default;
-    ~LogStream();
+    static RefPtr<LogStream> create() { return adoptRef(new LogStream()); }
 
     void setup(uint64_t pid, IPC::StreamServerConnectionHandle&&, LogStreamIdentifier, CompletionHandler<void(IPC::Semaphore& streamWakeUpSemaphore, IPC::Semaphore& streamClientWaitSemaphore)>&&);
 
 private:
+    LogStream() = default;
+    ~LogStream();
```

Comment:
Why is the destructor private? I guess this means LogStreams are never destroyed? Which is weird because the LogStream destructor does do work.

Reply:
Still private?

Reply:
Ah, good catch! Fixing now.